src/v2/Apps/Consign/Routes/MarketingLanding/Components/ReadMore.tsx

```diff
@@ -1,23 +1,37 @@
-import * as React from "react";
+import * as React from "react"
 import { Box, Column, GridColumns, Image, Text } from "@artsy/palette"
 import { RouterLink } from "v2/System/Router/RouterLink"
 import { useTracking } from "react-tracking"
-import { ContextModule, OwnerType, clickedArticleGroup } from "@artsy/cohesion"
+import {
+  ActionType,
+  ClickedArticleGroup,
+  ContextModule,
+  OwnerType,
+  PageOwnerType,
+} from "@artsy/cohesion"
 import { cropped } from "v2/Utils/resized"
 
+interface AnalyticsOptions {
```

Comment:
My suggestions/questions from https://github.com/artsy/force/pull/8859/files also apply here (using `Partial/Omit` or removing `const tracks` - let's apply the same solution in both places once we've decided how to proceed :)

Reply:
Thanks, Matt!! Just applied this (3724941)